clang/lib/StaticAnalyzer/Checkers/GenericTaintChecker.cpp

```diff
@@ -105,7 +105,8 @@ bool isStdin(SVal Val, const ASTContext &ACtx) {
   if (const auto *D = dyn_cast_or_null<VarDecl>(DeclReg->getDecl())) {
     D = D->getCanonicalDecl();
     // FIXME: This should look for an exact match.
-    if (D->getName().contains("stdin") && D->isExternC()) {
+    if (D->getName().contains("stdin") && D->hasExternalStorage() &&
```

Comment:
Could you change this to an exact match, following the FIXME above this line? Is there any reason for using "contains"?

Reply:
Done